CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 * Enchancement: Support @SentrySpan and @SentryTransaction on classes and interfaces. (#1243)
 * Enchancement: Do not serialize empty collections and maps (#1245)
 * Ref: Simplify RestTemplate instrumentation (#1246)
+* Ref: Optimize DuplicateEventDetectionEventProcessor performance (#1247).
```

Comment:
should also go under the `breaking change` section

Reply:
It's changing the algorithm that sits under the hood. I am not sure why this would be qualified as a breaking change.

Reply:
its a behavior change, there's a `bufferSize` now and there is the possibility to report the same exception twice, right?

Reply:
Unlikely, only if applications sends hundreds/thousands errors per second. Buffer size is hidden from the user. I can add it to breaking change if you like but I don't believe we should bump a version because of this.